src/classes/STG_PanelHealthCheck_CTRL.cls

```diff
@@ -460,13 +460,16 @@ public without sharing class STG_PanelHealthCheck_CTRL extends STG_Panel {
         checkMultiRecordTypeIds(Opportunity.sObjectType, STG_Panel.stgService.stgCon.Opp_RecTypes_Excluded_for_Payments__c, 
             'npe01__Contacts_And_Orgs_Settings__c', UTIL_Namespace.StrTokenNSPrefix('Opp_RecTypes_Excluded_for_Payments__c'), Label.stgNavDonations, Label.stgNavPayments);   
             
-        checkRecordTypeId(Opportunity.sObjectType, STG_Panel.stgService.stgRD.npe03__Record_Type__c, 
-            'npe03__Recurring_Donations_Settings__c', 'npe03__Record_Type__c', Label.stgNavRecurringDonations, Label.stgNavRecurringDonations); 
+        if (!RD2_EnablementService.isEnhancementEnabled) {
+            checkRecordTypeId(Opportunity.sObjectType, STG_Panel.stgService.stgRD.npe03__Record_Type__c, 
```

Comment:
@force2b , should Recurring Donations record type Id be verified regardless of enhancement?